resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{{trans('Laravel Quickstart - Basic')}}</title>
+    {{-- Fonts --}}
+    <link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
+
+    <!-- Scripts -->
+    <script src="{{ asset('js/app.js') }}" defer></script>
```

Comment:
```suggestion
    <script src="{{ mix('js/app.js') }}" defer></script>
```

dùng mix thì đưa vào mix luôn nhé k để trong asset b

Reply:
e dùng thử mix thì bị lỗi ko nhận css ạ